simp_config.rb

```diff
@@ -156,32 +152,32 @@ def getvboxnetworkname(network)
 require 'fileutils'
 
 workingdir = ARGV[0]
-testdir = ARGV[1]
-basedir = File.expand_path(File.dirname(__FILE__))
-json_tmp = basedir + "/simp.json.template"
+testdir    = ARGV[1]
+basedir    = File.expand_path(File.dirname(__FILE__))
+json_tmp   = basedir + "/simp.json.template"
 
 default_settings = {
-      'vm_description'      => 'SIMP-PACKER-BUILD',
-      'output_directory'    => "#{testdir}/OUTPUT",
-      'nat_interface'       => 'enp0s3',
-      'host_only_interface' => 'enp0s8',
-      'mac_address'         => 'aabbbbaa0007',
-      'firmware'            => 'bios',
-      'host_only_gateway'   => '192.168.101.1',
-      'domain'              => 'simp.test',
-      'puppetname'          => 'puppet',
-      'new_password'        => 'P@ssw0rdP@ssw0rd',
-      'fips'                => 'fips=0',
-      'disk_encrypt'        => '',
-      'big_sleep'           => '',
-      'headless'            => 'true',
-      'iso_dist_dir'        => '/net/ISO/Distribution_ISOs',
-      'root_umask'          => '0077'
+  'vm_description'      => 'SIMP-PACKER-BUILD',
+  'output_directory'    => "#{testdir}/OUTPUT",
+  'nat_interface'       => 'enp0s3',
+  'host_only_interface' => 'enp0s8',
+  'mac_address'         => 'aabbbbaa0007',
+  'firmware'            => 'bios',
+  'host_only_gateway'   => '192.168.101.1',
+  'domain'              => 'simp.test',
+  'puppetname'          => 'puppet',
+  'new_password'        => 'P@ssw0rdP@ssw0rd',
+  'fips'                => 'fips=0',
+  'disk_encrypt'        => '',
+  'big_sleep'           => '',
+  'headless'            => 'true',
+  'iso_dist_dir'        => '/net/ISO/Distribution_ISOs',
+  'root_umask'          => '0077'
 }
 
 # input packer.yaml and merge with default settings
 in_settings = YAML.load_file("#{testdir}/packer.yaml")
-settings = validate_settings(default_settings.merge(in_settings))
+settings    = validate_settings(default_settings.merge(in_settings))
```

Comment:
Seems like this spacing change was an arbitrary style change.  I'm not a fan of changing someone else's style, in the absence of style guides, because it unnecessarily increases number of lines to review.  It can also result in style wars.

Reply:
@lnemsick-simp  As with the other variables assignments, it lines up the white space so the equal signs (and their right-hand values) are vertically aligned.  This is consistent with the rest of the code.